docs/developer_guidelines.md

```diff
@@ -124,68 +125,79 @@ Resources:
 - [One Idea is One Commit] - Interesting committing strategy, comparable to
   atomic commits.
 
-Use the `gitlint` tool to check your commit messages. See
-[tools/gitlint/README.md] for more information.
+### Message style and convention
 
-### Message style
+We follow [Conventional Commits] with a minor deviation:
```

Comment:
did you want to add a link here or are these brackets only to stand out?

Reply:
I forgot to add a link.

Reply:
I forgot a link, I have added it now.